SocialCareCaseViewerApi/V1/Gateways/IDatabaseGateway.cs

```diff
@@ -42,7 +42,7 @@ public interface IDatabaseGateway
         PersonalRelationship CreatePersonalRelationship(CreatePersonalRelationshipRequest request);
         PersonalRelationship GetPersonalRelationshipById(long relationshipId);
         void DeleteRelationship(long relationshipId);
-
+        List<CaseStatus> GetCaseStatusesByPersonId(long personId);
```

Comment:
We could make the return type IEnumerable<CaseStatus>, there is no reason for the Gateway to decide the structure of the data returned.

Reply:
Changed from List to IEnumerable